### PR TITLE
Updating Supplementary Figure - Initial Parameter Sweep

### PR DESCRIPTION
**Pull request by gwaygenomics**
Here I add an updated supplementary figure - I added the unconverged DAE parameters for GTEX and TCGA. The figure was updated after chatting with @cgreene yesterday. Here it is below.

z_dimension_sweep_summary

The main change is in cefdbec; a minor change rotating text is in d87f3dc; the actual figure is updated in d78681a; the remaining figures are added (after rerunning the pipeline) in 5915bab

### CODE REVIEW

**Comment by gwaygenomics** [comment]
> What is this DAE converge?

In order to visualize the difference in hyperparameters, i needed to remove points (hyper parameter combinations) that appeared to fail to converge

> Is this running DAE and looking at the results before its "fully learned"?

Not sure if something can ever be "fully learned" - at least with these algorithms. But yeah, its running DAE but the combinations of parameters don't learn well.

> Not sure how you can tell when its converged? Interesting idea..

This is a good point. I can't tell that its converged really. All i can tell is that for the specific combinations of hyperparameters I chose, the loss function didn't really change all that much. For some combinations, I know that the models perform poorly - so I should avoid these. 

I think that the "converge" label in this case is then misleading - I will update the label! Thanks!

> is there a reason you didn't do this for the other methods?

The only other method that requires a hyperparameter sweep (for my current implementations) are the VAEs. The VAEs were remarkably stable for all hyperparameter combinations - the loss function was not impacted much by tweaking hyperparameters.
